### PR TITLE
#325 - images alt tags

### PR DESCRIPTION
**Pull request by AleksandraKozubal**
It should close #325

### CODE REVIEW

File: lang/en/alt.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "reference" => "reference from",
+    "about" => "Isometric illustration of AI theme with people interacting with technology elements.",
+    "career" => "Isometric illustration of people interacting with a large webpage, indicating digital content creation.",
+    "career_2" => "Isometric illustration of people interacting with large web pages and digital elements, depicting online learning and information access.",
+    "career_3" => "Isometric illustration of individuals programming with laptops, code snippets, and development icons.",
+    "home" => "Isometric illustration of people with digital marketing icons, including a magnifying glass, drone, and graphs.",
+    "policy" => "Isometric illustration of a person checking a list with oversized clipboard, magnifying glass, and clock.",
+    "software_engineer" => "Isometric illustration of two people coding with giant laptop and tech icons around.",
+    "404" => "",
+    "home_project" => "",
```
Comment (EwelinaSkrzypacz) [inline lang/en/alt.php:17]: :eyes:

Comment (AleksandraKozubal) [reply lang/en/alt.php]: updated c:

**Comment by AleksandraKozubal** [comment]
> 1. On page "Kariera" in section "Benefity" the icon next to text "Multisport" has no `aria hidden=true`
> 
> 2. There is lack of space between text "referencja autorstwa" and name of author
> 
> 3. Label for email in contact form is not associated with input

1,2,3 fixed

File: lang/en/alt.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "reference" => "reference from",
+    "about" => "Isometric illustration of AI theme with people interacting with technology elements.",
+    "career" => "Isometric illustration of people interacting with a large webpage, indicating digital content creation.",
+    "career_2" => "Isometric illustration of people interacting with large web pages and digital elements, depicting online learning and information access.",
+    "career_3" => "Isometric illustration of individuals programming with laptops, code snippets, and development icons.",
+    "home" => "Isometric illustration of people with digital marketing icons, including a magnifying glass, drone, and graphs.",
+    "policy" => "Isometric illustration of a person checking a list with oversized clipboard, magnifying glass, and clock.",
+    "software_engineer" => "Isometric illustration of two people coding with giant laptop and tech icons around.",
+    "404" => "Izometryczna ilustracja krytego basenu z leżakami, roślinami i fontanną.",
+    "home_project" => "Strona internetowa poświęcona tematyce medycznej zawierająca pasek wyszukiwania, katalog produktów i koszyk zakupów.",
```
Comment (EwelinaSkrzypacz) [inline lang/en/alt.php:17]: :eyes:

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**